.openshift-ci/gcp/delete-vm.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+deleteGCPVM() {
+    local GCP_VM_NAME="$1"
+    shift
+
+    [ -z "$GCP_VM_NAME" ] && \
+        echo "error: missing parameter GCP_VM_NAME" && return 1
+
+    success=false
+    if gcloud compute instances delete "$GCP_VM_NAME";
+    then
+        success=true
+        #break
+    fi
+
+    if test ! "$success" = "true"; then
+        echo "Could not delete instance."
+        return 1
+    fi
```

Comment:
Similar to the comment on `create-vm.sh`
```suggestion
    if ! gcloud compute instances delete "$GCP_VM_NAME";
    then
        echo "Could not delete instance."
        return 1
    fi
```